content/latest/deploy/kubernetes/helm-chart.md

```diff
@@ -192,6 +192,32 @@ REVISION  UPDATED                   STATUS    CHART           DESCRIPTION
 1         Fri Oct  5 09:04:46 2018  DEPLOYED  yugabyte-latest Install complete
 ```
 
+## Connect using YCQL client
+
+By default YugaByte helm will expose only the master ui endpoint via LoadBalancer. If you wish to expose YCQL, YEDIS and YSQL services via LoadBalancer for your app to use, you could do that via the following command.
```

Comment:
"via" is used twice in the last sentence. Lets change to:
 could do that via the following command -> you can do that as follows.